index.js

```diff
@@ -1,18 +1,44 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+const header = document.querySelector("#header");
+console.log("Here's your header:", header);
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "lightgreen";
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler);
+document.querySelector("div#profile img").src = traveler.photo;
+document.querySelector("div#profile img").alt = traveler.name;
+document.querySelector("div#profile h2").innerText = traveler.name;
+document.querySelector("div#profile em").innerText = traveler.nickname;
 
 
 /***** Deliverable 4 *****/
+function createAnimalLi(animal){ 
+    let ul = document.querySelector("#animals");
+
+    let li = document.createElement("li");
+    let p = document.createElement("p");
+    let img = document.createElement("img");
+    let a = document.createElement("a");
+
+    li.setAttribute('data-id', animal.id);
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```